src/communication/network_interface_sr.c

```diff
@@ -7347,6 +7347,12 @@ smmon_get_module_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request,
 
   if (module_index == 0)
     {
+      /* If there is no modules that registered in memory monitoring manager,
+       * just return */
+      if (MMON_MODULE_LAST == 0)
```

Comment:
What happens if `module_index != 0` and `MMON_MODULE_LAST == 0`? It seems that it could happen if we assume that `module_index ==0` and `MMON_MODULE_LSAT == 0` is possible.
```suggestion
      if (MMON_MODULE_LAST == 0)
```

Reply:
depending on https://github.com/CUBRID/cubrid/pull/4590#discussion_r1297888097

Reply:
I prevent to request to server at `memmon()`. Please check it.